test/py/explicit_module/runtest.sh

```diff
@@ -23,9 +23,12 @@ rm -rf gen-py
 ../../../compiler/cpp/thrift --gen py test1.thrift || exit 1
 ../../../compiler/cpp/thrift --gen py test2.thrift || exit 1
 ../../../compiler/cpp/thrift --gen py test3.thrift && exit 1  # Fail since test3.thrift has python keywords
+../../../compiler/cpp/thrift --gen py:enum shared_types.thrift || exit 1
+../../../compiler/cpp/thrift --gen py:enum test4.thrift || exit 1
```

Comment:
I wonder if we could also have tests without `enum` arg to make sure that it also works in that case.

Reply:
The change specifically impacts enum as it is behind gen_enum_, otherwise it just uses the integer value. I can add a test but it would just be the value 0, 1 in the map.
```cpp
if (gen_enum_) {
      t_enum_value* enum_val = ((t_enum*)type)->get_constant_by_value(int_val);
 --     out << type->get_name() << "." << enum_val->get_name();
 ++     out << type_name(type) << "." << enum_val->get_name();
}
```